.github/workflows/case-data-update-prod.yml

```diff
@@ -14,6 +14,7 @@ on:
     - cron: "0 5 * * *"
 
 jobs:
+  # NOTE: This job should bee kept in sync with the dev version.
```

Comment:
git pull 🐝
aka. 🐝 kept in sync

Reply:
oooops. but it is more fun that way

🐝 our guest, 🐝 our guest, put our workjobs to the test!